db/seeds.rb

```diff
@@ -20,7 +20,7 @@
 pull_toy = dog_shop.items.create(name: "Pull Toy", description: "Great pull toy!", price: 10, image: "http://lovencaretoys.com/image/cache/dog/tug-toy-dog-pull-9010_2-800x800.jpg", inventory: 32)
 dog_bone = dog_shop.items.create(name: "Dog Bone", description: "They'll love it!", price: 21, image: "https://img.chewy.com/is/image/catalog/54226_MAIN._AC_SL1500_V1534449573_.jpg", active?:false, inventory: 21)
 
-merchant_user = User.create!(name: "Johnny",
+merchant_user1 = bike_shop.users.create!(name: "Johnny",
```

Comment:
nice addition, I had trouble creating a bunch of `items` with factory bot in the seeds yesterday, but we should try to do add that too.